cirq/sim/simulator.py

```diff
@@ -203,7 +203,7 @@ def simulate(
         """Simulates the supplied Circuit.
 
         This method returns a result which allows access to the entire
-        wave function.
+        quantum computer's final state.
```

Comment:
quantum computer seems awkward here, since it is a simulator.  Should we say the simulator's final state?

Reply:
Good call. fixed here and elsewhere.